components/home/Featured.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import ScaffoldContainer from '../ScaffoldContainer';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    paddingLeft: theme.spacing(5),
+    paddingRight: theme.spacing(5),
+  },
+  heading: {
+    fontWeight: 'bold',
+    paddingTop: '2rem',
+    color: '#598fe0',
+  },
+  content: {
+    fontWeight: 'lighter',
+    color: '#598fe0',
+    paddingTop: '1.2rem',
+  },
+  callToAction: {
+    color: 'white',
+    background: '#598fe0',
+    width: '10vw',
+    height: '4vh',
+    marginTop: '2.5rem',
+  },
+  itemContainer: {
+    textAlign: 'left',
+    '&:last-of-type': {
+      textAlign: 'center',
+    },
+  },
+  imageContainer: {
+    textAlign: 'left',
+  },
+  mainContainer: {
+    '&:nth-of-type(2)': {
+      flexDirection: 'row-reverse',
+    },
+    '&:nth-of-type(2n+3)': {
+      flexDirection: 'row-reverse',
+    },
+    paddingBottom: '2rem',
+    '&:nth-of-type(1)': {
+      height: '35rem',
+      paddingBottom: '3rem',
+    },
+    '&:last-of-type': {
+      height: '30rem',
+    },
+  },
+}));
+
+function Featured(props) {
+  const { featuredContent } = props;
+
+  const classes = useStyles();
+
+  const headingSize = (img, length) => {
+    if (img) {
+      if (length > 36) return 12;
+      return 9;
+    }
+    return 12;
+  };
+
+  return (
+    <div className={classes.root}>
+      <ScaffoldContainer>
+        {featuredContent.map(item => (
+          <Grid key={item.heading} container alignItems="center" className={classes.mainContainer}>
```

Comment:
since you have a repeating group (line 75 - 96) for each featured content item, let's pull it into a component -- i.e. insead of `{featuredContent.map(item => (<Grid key=...` something like 
```
...
{featuredContent.map(item => (
    <FeaturedContentItem heading={item.heading} image={item.img} .../> )}
```
Let me know if that makes sense